packages/cozy-harvest-lib/src/components/DeleteAccountButton.jsx

```diff
@@ -38,14 +39,19 @@ export class DeleteAccountButton extends Component {
   render() {
     const { t } = this.props
     const { deleting } = this.state
+    const excludedButtonProptypes = {
+      ...DeleteAccountButton.propTypes,
+      ...getMutationsProptypes(accountsMutations),
+      ...i18nContextTypes
+    }
```

Comment:
Can this be done outside the render ?

Reply:
Why is the purpose of doing this outside the render?

Reply:
It works outside of the render so let's do this 👍

Reply:
>  Why is the purpose of doing this outside the render?

Things that can be done outside the render should be done outside the render since `render` is a hot path and is frequently called.